src/PerfView/SupportFiles/UsersGuide.htm

```diff
@@ -19,7 +19,7 @@
     </style>
 </head>
 <body>
-    <!--  ************************************************************************************* -->
+    <!- - ************************************************************************************* -->
```

Comment:
This seems unintentional as it breaks the comment?

Reply:
OOPS! Fixed!